pages/language_framework_overview.html

```diff
@@ -153,7 +153,7 @@ <h2>Text layout index</h2>
 
 <figure id="fig_layout_index">
 <img src="language_framework_overview-data/layout_index.png" alt="">
-<figcaption>A small section in the International text layout and typography index.</figcaption>
+<figcaption>A small section in the Language enablement index.</figcaption>
```

Comment:
Either title case it (`Language Enablement`) or lower case it (`language enablement`). The other occurrences are sentence start, hence the initial cap.

Reply:
If you look at the document title https://www.w3.org/TR/typography/ it is downstyled: ie. Language enablement index.  I don't have very strong views on whether or not to retain the downstyling, although if it is link text or highlighted or marked by punctuation in some other way it may be better to keep it.

Note however that if capitalisation _is_ applied, it needs to be applied to all 3 words in the title, not just "Language Enablement" if the title is being quoted (which probably is the case at least most of the time).

Reply:
It might be downstyled in the document itself, but it should be treated as a title elsewhere. Otherwise it is super confusing (c.f. this pull request!... ooh, I used a Latin abbreviation... yikes!!) Or we should mark "Language enablement index" with `<cite>`.

In any case, in English the normal behavior for titlecase is to capitalize the first word and all nouns, pronouns, adjectives, verbs and adverbs, but not articles, conjunctions or prepositions.

Reply:
I have added a link, so I think it's less ambiguous now. Are you OK with it, @aphillips?

See the figure caption in https://deploy-preview-445--i18n-drafts.netlify.app/pages/language_framework_overview.html#fig_layout_index for the preview.